metals/src/main/scala/scala/meta/internal/metals/Time.scala

```diff
@@ -1,13 +1,11 @@
 package scala.meta.internal.metals
 
-import java.util.concurrent.TimeUnit
-
 /**
  * Wrapper around `System.nanoTime` to allow easier testing.
  */
 trait Time {
   def nanos(): Long
-  final def millis(): Long = TimeUnit.NANOSECONDS.toMillis(nanos())
+  final def millis(): Long = System.currentTimeMillis()
```

Comment:
Beware: this is not related to the nanos method anymore, which I think it was the intention

Reply:
You should check how the `Time` trait is used for testing, since that seems to be the original intent.
As @gabro points out, the idea might be that `millis()` is coupled to `nanos()` for testing purposes, and the `Time` is just a commodity util over nanotimes...

If that's the case, I'd suggest renaming `millis()` to something more explicit like `asMillis()` and document that.
Or even better, create an additional `millisTimestamp()` that doesn't relates to `nanos()` in any way but explicitly states it's getting some form of system timestamp...
On top of that you could rename `nanos` -> `monotonicNanos`, `millis` -> `monotonicMillis`?

Just wild ideas

Reply:
Millis were always used as a timestamp and nanos for timing, which is correct. The only thing is that there is no actual relation between nanoTime and current time.

Fixed it in the tests.

Reply:
To be honest, I don't see the benefit of using nanos here.
There is one place where nanos are used directly: to calculate the [StatusBar priority](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/StatusBar.scala#L174) needed for sorting the statuses. I would argue that using milliseconds here should not be a noticeable change.

Therefore I would like to propose dropping the support for nanoseconds.

Reply:
Of course there is another solution - never ever using this method for creating timestamps, but then that is an additional complexity

Reply:
I think we can leave nanos for StatusBar priority. I will rename `millis` to `currentMillis` to indicate the difference.